src/test/java/com/epam/training/center/hw2/ex2/IndexPage2Test.java

```diff
@@ -0,0 +1,112 @@
+package com.epam.training.center.hw2.ex2;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class IndexPage2Test extends AbstractPage2{
+
+    protected String URL = "https://jdi-testing.github.io/jdi-light/index.html";
+
+    @Test(priority = 1, description = "open page and assert browser title")
+    public void browserTitleText() {
+        driver.get(URL);
+        Assert.assertEquals(driver.getTitle(), "Home Page");
+    }
+    //input login and password, assert user name
+    @Test(priority = 2)
+    public void inputLoginDataTest() {
+        WebElement userIcon = driver.findElement(By.id("user-icon"));
+        userIcon.click();
+        WebElement loginField = driver.findElement(By.id("name"));
```

Comment:
Pls create method login(LoginUser loginUser) in BaseTest, where LoginUser - enum which contains login, password and username